install/requirements.txt

```diff
@@ -10,12 +13,15 @@ Jinja2==2.10
 macholib==1.11
 MarkupSafe==1.0
 pefile==2018.8.8
+pycparser==2.18
+pycryptodome==3.6.6
 PyInstaller==3.4
+PyNaCl==1.2.1
 PyQt5==5.11.2
 PyQt5-sip==4.19.12
 PySocks==1.6.8
 requests==2.19.1
-sip==4.19.8
```

Comment:
Got a warning when packaging on my Windows 'WARNING: hidden import 'sip' not found'. Not sure if important